index.js

```diff
@@ -88,16 +91,26 @@ class Geocoder extends Component {
         })
       }
     } else {
-      search({
+      const autocompleteQuery = {
         apiKey,
         boundary,
         focusPoint,
         text: input
-      }).then((geojson) => {
+      }
+      const autocompleteQueryKey = stableStringify(autocompleteQuery)
```

Comment:
The reason I'm using stable stringify is that the following produce 2 different results:

```js
console.log(JSON.stringify({a:1,b:2}))  // "{"a":1, "b":2}"
console.log(JSON.stringify({b:1,a:2}))  // "{"b":2, "a":1}"
```